### DISCUSSION TITLE
Here is the French translation (based upon latest en-US) for "GetMyIP_0.12.3"

### DISCUSSION THREAD

**Discussion by logounet** [Timthreetwelve/GetMyIP#128] [General]
Hello,

Here is the French translation (based upon latest en-US) for "GetMyIP_0.12.3"

See attached file

[Strings.fr-FR.xaml.zip](https://github.com/user-attachments/files/18059347/Strings.fr-FR.xaml.zip)

Best regards

**Comment by Timthreetwelve** [comment]
@logounet 

Thank you!
